src/luarocks/core/cfg.lua

```diff
@@ -287,12 +287,12 @@ local function make_defaults(lua_version, target_cpu, platforms, home)
 
       defaults.makefile = "Makefile.win"
       defaults.variables.PWD = "echo %cd%"
-      defaults.variables.MAKE = "nmake"
-      defaults.variables.CC = "cl"
-      defaults.variables.RC = "rc"
-      defaults.variables.LD = "link"
-      defaults.variables.MT = "mt"
-      defaults.variables.AR = "lib"
+      defaults.variables.MAKE = os.getenv("MAKE") or "nmake"
+      defaults.variables.CC = os.getenv("CC") or "cl"
+      defaults.variables.RC = os.getenv("WINDRES") or "rc"
+      defaults.variables.LD = os.getenv("CC") or "link"
```

Comment:
should this not use the env var

Reply:
ah, good catch! — I think for platforms where you do get a distinct linker by default it should be fine to keep the platform's default expected linker variable (is it usually `LD` or `LINK` on Windows with MSVC? I don't know)

Reply:
no idea, i've removed the getenv for it

Reply:
From what I can quickly guess from the web, looks like the most likely variable in that case would be "LINK". I'll merge this PR and add that!